### PR TITLE
Copy Compose YAML to Clipboard

### PR DESCRIPTION
**Pull request by dharmamike**

### CODE REVIEW

File: app/assets/javascripts/jquery.compose_export_dialog.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        headers: {
+          'Accept': 'application/json'
+        },
+        url: composeUrl
+      })
+        .done(function(response) {
+          composeDialog = base.initiateDialog(response.compose_yaml);
+          composeDialog.dialog('open');
+
+          $('.clipboard-copy').attr('data-clipboard-text', $('#composeYaml').text());
```
Comment (alexwelch) [inline app/assets/javascripts/jquery.compose_export_dialog.js]: The OCD in me does not like all the hard DOM dependencies, but I have a feeling you don't want to look at these lines of code anymore, :wink:

**Comment by alexwelch** [comment]
:+1:

File: app/assets/javascripts/jquery.compose_export_dialog.js (hunk 1 above)
Comment (dharmamike) [inline app/assets/javascripts/jquery.compose_export_dialog.js]: the alternative is to add these things to the options or defaultOptions object?

Comment (alexwelch) [inline app/assets/javascripts/jquery.compose_export_dialog.js]: yeah

**Comment by rheinwein** [comment]
:+1:

**Pull request merged**